src/io/flutter/editor/WidgetIndentsHighlightingPass.java

```diff
@@ -135,13 +135,13 @@ boolean updateSelected(@NotNull Editor editor, @NotNull RangeHighlighter highlig
       final Document doc = highlighter.getDocument();
       final int caretOffset = carat.getOffset();
 
-      if (startOffset >= doc.getTextLength()) {
+      if (startOffset < 0 || startOffset >= doc.getTextLength()) {
         return setSelection(false);
       }
 
       final int endOffset = highlighter.getEndOffset();
 
-      int off = startOffset;
+      int off;
```

Comment:
was this change intentional? Seems accidental.

Reply:
It was - IntelliJ helpfully told me that `off` was assigned to again before it was first read, so this initial value was never used.